src/motion-queue.h

```diff
@@ -90,6 +111,11 @@ class MotionQueue {
 
   // Shutdown. If !flush_queue: immediate, even if motors are still moving.
   virtual void Shutdown(bool flush_queue) = 0;
+
+  // Return the pointer to the status register.
+  virtual void GetStatus(struct QueueStatus *status) = 0;
+
+  PositionChangeCallback *callbacks;
```

Comment:
Never put a member accessible publically. Make a setter if you want to set it.
Describe what it does.